### ISSUE TITLE
Why calculate loss on unscaled predictions while training?

### ISSUE THREAD

**Issue by veds12** [jdb78/pytorch-forecasting#649]
Hi @jdb78. I understand that the error metrics should be calculated in the original un-scaled space while evaluating. But could you please explain why it won't make sense to backpropagate on loss calculated in scaled space (i.e. calculating errors during test time)? Calculating loss in original space might lead to problems like exploding loss (going to NaN); especially in series where the values are of the order of ten thousand or above. I didn't face this problem (exploding losses and exploding gradients) in normal training but found it to be very prominent in applications like Meta-Learning (since it involves calculating of second-order gradients).
